### PR TITLE
TASK SIO-3787 add available entities

### PR DESCRIPTION
**Pull request by vss414**

### CODE REVIEW

File: locations.go [hunk 1]
```diff
@@ -19,6 +19,13 @@ type Location struct {
 	IsDefault        bool              `json:"is_default"`
 	IsVisible        bool              `json:"is_visible"`
 	ComputeResources []ComputeResource `json:"compute_resources"`
+	AvailablePlans   []ShortPlan       `json:"available_plans"`
+}
+
+// ShortLocation represents short data of location.
```
Comment (oneumyvakin) [inline locations.go:25]: Please add an example, what is 'short' location.

**Review by oneumyvakin — APPROVED** [review]

File: locations.go [hunk 2]
```diff
@@ -19,6 +19,13 @@ type Location struct {
 	IsDefault        bool              `json:"is_default"`
 	IsVisible        bool              `json:"is_visible"`
 	ComputeResources []ComputeResource `json:"compute_resources"`
+	AvailablePlans   []ShortPlan       `json:"available_plans"`
+}
+
+// ShortLocation represents only ID and name of location.
+type ShortLocation struct {
```
Comment (dshemin) [inline locations.go:26]: Please make a different pull request and move it to the place where the full version of Locations was placed.
